### ISSUE TITLE
GCN apply for my own dataset but training_acc is low

### ISSUE THREAD

**Issue by zq1335030905** [tkipf/kegra#56]
my data is supervised, I don‘t know whether it will influence the result, and my training_acc is 0.0911,but val_acc is alwasy 0. and trainning_loss is decresing but val_loss is incresing. I don't know how to fix it , if anyone can help me , thank you a lot.

**Comment by xiyouxbr** [comment]
What is the format of your data set?

**Comment by zq1335030905** [comment]
my_Data.content:
number_id first_x first_y second_x second_y ... fourteen_x fourteen_y label
(first_x,first_y) is the coordinates of first point. X and y are all float number not 0/1.
my_Data.cites:
first_number_id second_number_id

And my data set is not for semi-supervised but for supervised. I don't know whether it will influence my result.  Is gcn like clustering for semi-supervised ? Thanks for your answering.

**Comment by xiyouxbr** [comment]
Hello, gcn network is the supervised network.
The input given by the author is a word. The author means that in the .content file, the word is represented as 1 in the paper, and 0 if it is not present in the paper. No float type used
Looking at your own data set, you may have a problem with your data set design, and then use the author's gcn model to have low accuracy.


xie borun
xiyouxbr@outlook.com
 
From: lzq133503
Date: 2020-03-28 13:55
To: tkipf/keras-gcn
CC: xiyouxbr; Comment
Subject: Re: [tkipf/keras-gcn] GCN apply for my own dataset but training_acc is low (#56)
my_Data.content:
number_id first_x first_y second_x second_y ... fourteen_x fourteen_y label
(first_x,first_y) is the coordinates of first point. X and y are all float number not 0/1.
my_Data.cites:
first_number_id second_number_id
And my data set is not for semi-supervised but for supervised. I don't know whether it will influence my result. Is gcn like clustering for semi-supervised ? Thanks for your answering.

**Comment by zq1335030905** [comment]
But my feature of a node cannot be represented by 0/1 , how can I solve this problem. If I use 0/1 to represent a node, that will be a parse matrix about 320*96 but there are 28 that value 1 and other is 0 . It may increase my calculation . ... If you have some advice, thanks very much.
